### PR TITLE
disabling db debug logging

### PR DESCRIPTION
**Pull request by jmervine**
@reidmix @joshuatobin

### CODE REVIEW

File: config/environments/production.rb [hunk 1]
```diff
@@ -47,6 +47,10 @@
   # when problems arise.
   config.log_level = :debug
 
+  config.after_initialize do
+    ActiveRecord::Base.logger.level = 1
```
Comment (reidmix) [inline config/environments/production.rb]: Woot.  I'd say use `Logger::INFO` (which is 1) because it's more expressive.

Comment (joshuatobin) [inline config/environments/production.rb]: Can we make this a config var instead and default it to `Logger::INFO`?

Comment (jmervine) [inline config/environments/production.rb]: +1, this will allow us to easily enable debug logging in staging and such.

**Comment by joshuatobin** [comment]
👍

File: config/environments/development.rb [hunk 2]
```diff
@@ -56,6 +56,10 @@
     password:       Config.smtp_password
   }
 
+  config.after_initialize do
+    ActiveRecord::Base.logger.level = ("Logger::#{Config.log_level.upcase}".constantize rescue 0)
```
Comment (reidmix) [inline config/environments/development.rb]: I'd say, even with the `rescue 1/0` use the expressive names `Logger::Debug`/`Logger::Info`

Comment (jmervine) [inline config/environments/development.rb]: Okay, fine. xP  I was just  trying to keep the lines shorter.

**Pull request merged**